tracer/src/Datadog.InstrumentedAssemblyGenerator/ProfilerMetadataImporter.cs

```diff
@@ -47,6 +47,8 @@ private void ImportInstrumentedAssemblyMetadata()
                 return;
             }
 
+            // TODO: Add parent type (declaring type) to be able to identify two nested types with the same name -- see MetadataNameParser.Parse
+            // https://datadoghq.atlassian.net/browse/DEBUG-1029
```

Comment:
Please remove JIRA link

Reply:
Fixed in 871a652d9b150d63443d87a542e34d4643145adf